### PR TITLE
Update etcd config description of kubeadm HA doc

### PR DESCRIPTION
**Pull request by takmatsu**
- Fix url from http to https
- Add etcd-listen-ip description

### CODE REVIEW

**Comment by takmatsu** [comment]
@k8s-ci-robot pls recheck

**Comment by Bradamant3** [comment]
@kubernetes/sig-cluster-lifecycle-pr-reviews PTAL -- note that this PR is against master, which means if we get it in before the 1.11 release, it ships only with the 1.10 docs version.

/assign

File: content/en/docs/setup/independent/high-availability.md [hunk 1]
```diff
@@ -244,14 +244,14 @@ Please select one of the tabs to see installation instructions for the respectiv
...
 
      [Install]
      WantedBy=multi-user.target
      EOF
      ```
 
-     Make sure you replace `<etcd0-ip-address>`, `<etcd1-ip-address>` and `<etcd2-ip-address>` with the appropriate IPv4 addresses. Replace `<name>` with the name of this etcd member. Modify the values of `--listen-client-urls`, `--advertise-client-urls`, `--listen-peer-urls` and `--initial-advertise-peer-urls` if needed. Replace `<etcd0>`, `<etcd1>` and `<etcd2>` with real hostnames of each machine. These machines must be able to reach every other using DNS or make sure that records are added to `/etc/hosts`.
+     Make sure you replace `<etcd0-ip-address>`, `<etcd1-ip-address>` and `<etcd2-ip-address>` with the appropriate IPv4 addresses. Replace `<name>` with the name of this etcd member. Modify the values of `--listen-client-urls`, `--advertise-client-urls`, `--listen-peer-urls` and `--initial-advertise-peer-urls` like if in etcd0 node, `<etcd-listen-ip>` is `<etcd0-ip-address>`. Replace `<etcd0>`, `<etcd1>` and `<etcd2>` with real hostnames of each machine. These machines must be able to reach every other using DNS or make sure that records are added to `/etc/hosts`.
```
Comment (neolit123) [inline content/en/docs/setup/independent/high-availability.md]: what do you mean by:
> `--initial-advertise-peer-urls` like if in etcd0 node, `<etcd-listen-ip>` is `<etcd0-ip-address>`

needs grammar fixes and better context. possibly two sentences.

> These machines must be able to reach every using DNS

`reach each other using DNS` ?

**Comment by timothysc** [comment]
This should be deprecated by @chuckha 's recent merge that overhauls the HA documentation.

**Comment by timothysc** [comment]
PSA this document change would *only* apply towards 1.10.

**Comment by Bradamant3** [comment]
@takmatsu if you could get the requested content changes in by next Monday, June 25, we can get this PR merged into the current master (1.10 docs). After that, it'll be too late. lmk if you have any questions about how to reword. I can help copyedit if need be, but technical clarification needs to come first. Thanks!

**Comment by Bradamant3** [comment]
(The comment about being deprecated is not strictly true: the new version of this doc is for version 1.11  and later only)

**Pull request merged**